src/maintainer/maintainer_faq.rst

```diff
@@ -164,3 +164,28 @@ FAQ
     build:
       script_env:
         - QT_QPA_PLATFORM=offscreen
+
+
+.. _mfaq_contact_core:
+
+:ref:`(Q) <mfaq_contact_core>` **How can I contact conda-forge/core?**
+
+  When in an issue or PR, you can contact `conda-forge/core <https://conda-forge.org/docs/orga/governance.html#teams-roles>`_ by simply mentioning ``@conda-forge/core`` in a comment.
+  If you don't receive an an answer after a couple of days, feel free to reach out to us via the public `gitter <https://gitter.im/conda-forge/conda-forge.github.io>`_ channel.
+
+  .. note::
+
+    Due to a GitHub limitation, this is disabled for new members.
```

Comment:
I recall the bot had a [command](https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-ping-team) to ping teams on behalf of new users. Is that still around? Does it support pinging core directly?

Reply:
It just pings the team of the feedstock which is basically the same as core on `staged-recipes` but not on other repos.

Reply:
Got it. I wonder if the rest of Core thinks it's a good idea to add a new command for this use case?

Reply:
you can already do @conda-forge-admin, please ping conda-forge/core

Reply:
Ahh ok then that should probably be added to https://conda-forge.org/docs/maintainer/infrastructure.html#admin-web-services - I'll update it.

Reply:
I cannot find anything in https://github.com/conda-forge/conda-forge-webservices/blob/main/conda_forge_webservices/commands.py - are you sure?